apps/meteor/server/startup/serverRunning.js

```diff
@@ -72,8 +72,8 @@ Meteor.startup(function () {
 			exitIfNotBypassed(process.env.BYPASS_NODEJS_VALIDATION);
 		}
 
-		if (!semver.satisfies(semver.coerce(mongoVersion), '>=3.6.0')) {
-			msg += ['', '', 'YOUR CURRENT MONGODB VERSION IS NOT SUPPORTED,', 'PLEASE UPGRADE TO VERSION 3.6 OR LATER'].join('\n');
+		if (!semver.satisfies(semver.coerce(mongoVersion), '>=4.2.0')) {
+			msg += ['', '', 'YOUR CURRENT MONGODB VERSION IS NOT SUPPORTED,', 'PLEASE UPGRADE TO VERSION 4.2 OR LATER'].join('\n');
```

Comment:
```suggestion
			msg += ['', '', 'YOUR CURRENT MONGODB VERSION IS NOT SUPPORTED,', 'PLEASE UPGRADE TO VERSION 4.4 OR LATER'].join('\n');
```
maybe we should directly ask to use 4.4

Reply:
else basically they'll update to 4.2 and see the deprecation warning. better ask them to start with the supported version imo

Reply:
No, because 4.2 is the last "supported" version (even tho it's deprecated. So customers can live there until we drop support (as many would probably live with 3.6 for a while, hehe)

Reply:
oh no I commited already 🙈 

But, even though that is supported, the message basically asks them to start using 4.2, which is not good. I get the technicality of it but for new deployments this is problematic

Reply:
maybe we mention the fact that 4.2 is "deprecated" explicitely?

Reply:
Well, good enough. That way less ppl will know 4.2 is still around 🙈

Reply:
should we go the "supported yet deprecated" way? i do kinda agree with your point too - since 4.2 is supported still.

Reply:
nvm it won't exit out - 4.4 message is good enough 🙊 approving